api/controllers/game/get-history.js

```diff
@@ -0,0 +1,52 @@
+const GameStatus = require('../../../utils/GameStatus.json');
+
+module.exports = async function getHistory(req, res) {
+  const userId = req.session.usr;
+
+  const sortBy = req.query.sortBy || 'createdAt';
```

Comment:
We should sanitize the input to prevent SQL injection shenanigans
```suggestion
  const allowedSortColumns = [ 'createdAt', 'winner', 'isRanked' ];
  const sortBy = allowedSortColumns.find((col) => col === req.query.sortBy) ?? 'createdAt';
```

Reply:
Changed it